adoc/release-2.1.0.adoc

```diff
@@ -0,0 +1,113 @@
+// Start attribute entry list (Do not edit here! Edit in entities.adoc)
+ifdef::env-github[]
+:suse: SUSE
+:current-year: 2020
+:product: {suse} Cloud Application Platform
+:version: 2.0
+:rn-url: https://www.suse.com/releasenotes
+:doc-url: https://documentation.suse.com/suse-cap/2
+:deployment-url: https://documentation.suse.com/suse-cap/2.0/single-html/cap-guides/#part-cap-deployment
+:caasp: {suse} Containers as a Service Platform
+:caaspa: {suse} CaaS Platform
+:ostack: OpenStack
+:cf: Cloud Foundry
+:kubecf: KubeCF
+:k8s: Kubernetes
+:scc: {suse} Customer Center
+:azure: Microsoft Azure
+:aks: Azure {k8s} Service
+:aksa: AKS
+:aws: Amazon Web Services
+:awsa: AWS
+:eks: Amazon Elastic Container Service for Kubernetes
+:eksa: Amazon EKS
+:gke: Google Kubernetes Engine
+:gkea: GKE
+:mysql: MySQL
+:mariadb: MariaDB
+:postgre: PostgreSQL
+:redis: Redis
+:mongo: MongoDB
+:ng: NGINX
+endif::[]
+// End attribute entry list
+
+[id='sec.2_1_0']
+=== Release 2.1.0, October 2020
+
+[id='sec.2_1_0.new']
+==== What Is New?
+* {kubecf} has been updated to version 2.5.8:
+** For a full list of features and fixes see https://github.com/cloudfoundry-incubator/kubecf/releases/tag/v2.5.8
```

Comment:
These notes have to include a roll up of all features and fixes in KubeCF releases since CAP 2.0.1. Add a list of links for all KubeCF releases since then. (this should probably go in a Component Versions section of the relnotes)

Reply:
I'll dig into this, leading back to KubeCF 2.2.3+. Do we need to cover cf-operator changes too?

Reply:
I have added links to all the releases since 2.2.3 but I don't think it looks particularly good or makes much sense because it's very easy to find all the releases from the link to the newest version. If we somehow want to include this, it would maybe be better to write up a short list of the main features or fixes from the releases since 2.2.3 and just include a link to the current version's changelog. What do you think?

Reply:
I'm not particularly concerned if we don't have links to each release, but we have to declare in the release notes (as you have) that it's an upgrade from 2.2.3 so that people can find the details if they need them. Do whatever you think would be useful for a platform operator. A summary of main features and fixes directly in the release notes would probably be most helpful. I'll leave the decision to you.